### DISCUSSION TITLE
Legal usage in germany - Imprint and copy right?

### DISCUSSION THREAD

**Discussion by Tockra** [PrivateBin/PrivateBin#1374] [General]
Hello,

maybe somebody here is familar with german or european law.
Is it possible to run a private bin application on a german domain in professional context?
Do I need a imprint? Because it is not a private used application I'm not sure about this.
Is there a way to add a imprint?

What is if somebody uses the public reachable server and abuses it to upload copy right protected stuff there?

T

**Comment by elrido** [comment]
Given that IANAL and clearly TINLA and not being located in Germany, I won't comment on the use of an imprint.

I can comment, though, that in many jurisdictions, when someone is abusing your public instance (which is not a question of "if", but just a matter of "when") you will be expected to take the offending content down ASAP. The point of PrivateBin is simply to provide you with some plausible deniability on why you didn't filter out the offending content before it got published. If that would actually hold as a defense in court in your particular jurisdiction is of course an entirely different topic.

Maintaining the demo instance of this project for some years, we had varying levels of take-down-requests (by lawyers, copyright-holders, activists and in rare cases law enforcement agencies) and found that limiting the duration of the pastes can serve to reduce such noise.

**Comment by Tockra** [comment]
Does privatebin somehow allows me to add a imprint easily?

**Reply by rugk** [reply]
We've had such requests for some time, like in https://github.com/PrivateBin/PrivateBin/issues/267, which eventually were implemented: https://github.com/PrivateBin/PrivateBin/pull/681

You can just adjust the `info` property and put in HTML there possibly linking to your imprint, data privacy policy or whatever legal things or other things you want to link: https://github.com/PrivateBin/PrivateBin/blob/3cba170f3255de21bbebb77f6c565519ef33e8c1/cfg/conf.sample.php#L55

I have added [the doc for this in the wiki](https://github.com/PrivateBin/PrivateBin/wiki/Configuration#info) now, yes it was actually missing. :upside_down_face: 

Obviously, if you want to have more control over the styling etc. you could adjust the [template](https://github.com/PrivateBin/PrivateBin/wiki/Configuration#template) and copy one and use your own one then…
